apidef/api_definitions.go

```diff
@@ -593,6 +593,7 @@ type APIDefinition struct {
 	CustomMiddleware           MiddlewareSection      `bson:"custom_middleware" json:"custom_middleware"`
 	CustomMiddlewareBundle     string                 `bson:"custom_middleware_bundle" json:"custom_middleware_bundle"`
 	CacheOptions               CacheOptions           `bson:"cache_options" json:"cache_options"`
+	RespectKeyExpiration       bool                   `bson:"respect_key_expiration" json:"respect_key_expiration"`
```

Comment:
I think it require a better name. Like: `remove_keys_after_expiration`

Reply:
If session lifetime is bigger, it doesn’t remove after expiration. It waits for expiration to be done

Reply:
In this case name it like: `session_lifetime_respect_key_expiration`, or similar.